src/controllers/authentication.js

```diff
@@ -472,8 +472,6 @@ authenticationController.logout = async function (req, res, next) {
 
 		await destroyAsync(req);
 		res.clearCookie(nconf.get('sessionKey'), meta.configs.cookie.get());
-		req.uid = 0;
-		req.headers['x-csrf-token'] = req.csrfToken();
```

Comment:
These two lines were added back when we were trying `no-refresh-login`. No longer required.